mailhog/mailhog/extension.py

```diff
@@ -0,0 +1,107 @@
+import os
+from typing import TYPE_CHECKING, Optional
+
+from localstack.extensions.api import Extension, http
+
+if TYPE_CHECKING:
```

Comment:
probably missing something, why is this a conditional import?

Reply:
all imports defined at module level are imported when the plugin is resolved. the deferred import should be the standard in extensions to help with loading times.
that said, since extensions are not really loaded lazily it's pretty pointless. i'll leave it in there for consistency though and add a comment.